src/spikeinterface/curation/curation_model.py

```diff
@@ -0,0 +1,360 @@
+    )
+    format_version: str = Field(..., description="Version of the curation format")
+    unit_ids: List[Union[int, str]] = Field(..., description="List of unit IDs")
+    label_definitions: Optional[Dict[str, LabelDefinition]] = Field(
+        default=None, description="Dictionary of label definitions"
+    )
+    manual_labels: Optional[List[ManualLabel]] = Field(default=None, description="List of manual labels")
+    removed: Optional[List[Union[int, str]]] = Field(default=None, description="List of removed unit IDs")
+    merges: Optional[List[Merge]] = Field(default=None, description="List of merges")
+    splits: Optional[List[Split]] = Field(default=None, description="List of splits")
+
+    @field_validator("label_definitions", mode="before")
+    def add_label_definition_name(cls, label_definitions):
+        if label_definitions is None:
+            return {}
+        if isinstance(label_definitions, dict):
+            label_definitions = dict(label_definitions)
+            for key in list(label_definitions.keys()):
+                if isinstance(label_definitions[key], dict):
+                    label_definitions[key] = dict(label_definitions[key])
+                    label_definitions[key]["name"] = key
+            return label_definitions
+        return label_definitions
+
+    @classmethod
+    def check_manual_labels(cls, values):
+
+        unit_ids = list(values["unit_ids"])
+        manual_labels = values.get("manual_labels")
+        if manual_labels is None:
+            values["manual_labels"] = []
+        else:
+            manual_labels = list(manual_labels)
+            for i, manual_label in enumerate(manual_labels):
+                manual_label = dict(manual_label)
+                unit_id = manual_label["unit_id"]
+                labels = manual_label.get("labels")
+                if labels is None:
+                    labels = set(manual_label.keys()) - {"unit_id"}
+                    manual_label["labels"] = {}
+                else:
+                    manual_label["labels"] = {k: list(v) for k, v in labels.items()}
+                for label in labels:
+                    if label not in values["label_definitions"]:
+                        raise ValueError(f"Manual label {unit_id} has an unknown label {label}")
+                    if label not in manual_label["labels"]:
+                        if label in manual_label:
+                            manual_label["labels"][label] = list(manual_label[label])
+                        else:
+                            raise ValueError(f"Manual label {unit_id} has no value for label {label}")
+                if unit_id not in unit_ids:
+                    raise ValueError(f"Manual label unit_id {unit_id} is not in the unit list")
+                manual_labels[i] = manual_label
+            values["manual_labels"] = manual_labels
+        return values
+
+    @classmethod
+    def check_merges(cls, values):
```

Comment:
The structure of split_data should be desribe by mode here.
The split_data type is unclear.